news/gitignore.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+*  <news item>
+
+**Changed:**
+
+* Updated `.gitignore` to include IDE/editor config
```

Comment:
```suggestion
* Updated `.gitignore` to include IDE/editor config. (#1413)
```

I like having a backreference to the PR that introduced the change (and/or the issue being fixed) because the history gets lost (a bit) in the rever release process. This way we can just go to it (and the context discussion) from the release notes. Thanks!

Reply:
Fixed in 22d4d330360022b89c475c6760e6f69173d9f02b